### PR TITLE
Added documentation for KLEE options to output statistics

### PR DESCRIPTION
**Pull request by andreamattavelli**
Added section to KLEE options about statistics, including KLEE PR #410 (Add option to dump statistics every n instructions).

### CODE REVIEW

File: docs/options.md [hunk 1]
```diff
@@ -85,3 +85,20 @@ To change the entry point you can use the option `-entry-point=FUNCTION_NAME`, w
 ## Calls to `klee-assume`
 
 By default, KLEE will report an error if the assumed condition is infeasible. The option `-silent-klee-assume` can be used to sliently terminate the current path exploration in such cases.
+
+## Statistics
+
+By default, KLEE generates two files containing statistics concerning the code exploration:
+
+* **run.stats**: This is a text file containing various statistics emitted by KLEE. While this file can be inspected manually, you should use the [klee-stats]({{site.baseurl}}/docs/tools) tool for that.
+* **run.istats**: This is a binary file containing global statistics emitted by KLEE for each line of code in the program. The file can be read with the callgring tool.
```
Comment (MartinNowack) [inline docs/options.md]: "binary file" -> "text file" using "callgrind format".
Those files can be read with frontends, which are able to read it (e.g. https://kcachegrind.github.io/)

File: docs/options.md [hunk 2]
```diff
@@ -85,3 +85,20 @@ To change the entry point you can use the option `-entry-point=FUNCTION_NAME`, w
...
+
+There are several option to modify how KLEE outputs statistics:
```
Comment (MartinNowack) [inline docs/options.md]: "options"

File: docs/options.md [hunk 3]
```diff
@@ -85,3 +85,20 @@ To change the entry point you can use the option `-entry-point=FUNCTION_NAME`, w
...
+
+* `-stats`                             - Enable statistics output from program (*available with Asserts*)
```
Comment (MartinNowack) [inline docs/options.md]: "available with Asserts" -> default=on

File: docs/options.md [hunk 4]
```diff
@@ -85,3 +85,20 @@ To change the entry point you can use the option `-entry-point=FUNCTION_NAME`, w
...
+* `-output-stats`                      - Write running stats trace file (*default=on*)
+* `-output-istats`                     - Write instruction level statistics in callgrind format (*default=on*)
```
Comment (MartinNowack) [inline docs/options.md]: "callgrind"

File: docs/options.md (hunk 1 above)
Comment (andreamattavelli) [inline docs/options.md]: Thanks. I just copied the message in klee --help

**Comment by MartinNowack** [comment]
Beside the comments, looks good to me.

What do you think about putting the instruction logging there as well?

File: docs/options.md (hunk 1 above)
Comment (MartinNowack) [inline docs/options.md]: Oh, ok - that's a pity. We should update the KLEE code as well.

**Comment by andreamattavelli** [comment]
> What do you think about putting the instruction logging there as well?

Sure, but in another PR

**Pull request merged**